README.md

```diff
@@ -108,7 +108,8 @@ To run tests, run the following command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Decode message]**
+- [ ] **[Dynamic]**
+- [] **[Responsive]
```

Comment:
[OPTIONAL]
- [ ] I kindly suggest you correct this line of code in your readme so that it is displayed correctly, see the below code example for guidance.
```readme.md
- **[Responsive]**
```